<a href="https://colab.research.google.com/github/JosephHobbs9292/Project-Two/blob/main/Project_2_Data(1).ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [1]:
## Importing Libraries

import pandas as pd


In [2]:
## Mounting Drive
from google.colab import drive
drive.mount('/content/drive')

Mounted at /content/drive


In [3]:
df_graduation = pd.read_csv("/content/Project One Data - Final Data.csv")
df_graduation.head()

Contact ID  Zip Code                                  High School  \
0  0031G00000q9mag       NaN                                          NaN   
1  0031G00000q9mal       NaN                                          NaN   
2  0031G00000q9mb0       NaN                                          NaN   
3  0031G00000q9mfW       NaN                                          NaN   
4  0031G00000q9nLu   85035.0  Western School of Science and Technology HS   

  Gender  Cohort  Graduated                               College  \
0    NaN  2019.0          0           NORTHERN ARIZONA UNIVERSITY   
1    NaN  2019.0          0              Arizona State University   
2    NaN  2020.0          0           NORTHERN ARIZONA UNIVERSITY   
3   Male  2014.0          0              Arizona State University   
4    NaN  2021.0          0  SAE Institute of Technology-New York   

  College Location Public/Private  Number of Sessions  
0         In State         Public                  22  
1         In State         Public                  22  
2         In State         Public                  25  
3         In State         Public                   4  
4     Out of State        Private                  42

In [4]:
df_graduation.shape

(5644, 10)

--- 
## Data Explanation

---
1. Be A Leader Foundation 
2. The data show wether or not a student graduated from college with some relevant fields to their graduation success
3. The target is the graduation column
4. Each row is one student and their relevant data such as zip or high school
5. This is a binary classification problem focusing on wether or not a student graduated from college
6. There are 8 features in the dataset: Zip code, high school, gender, cohort, college, public or private college, number of working sessions
7. 5644
8. There is some missing data and perhaps inconsistent naming amongst the colleges that will need to be cleaned up. 